import_export/resources.py

```diff
@@ -905,6 +906,12 @@ def get_import_id_fields(self):
         """
         return self._meta.import_id_fields
 
+    def get_import_id_column_names(self):
```

Comment:
Could this method name be improved? Maybe “get_import_id_field_or_column_names”?

Reply:
My understanding is: it's always returning a list of 'column names' (even if they aren't from the `column_name` attribute), as this is what we're searching for in the data source headers. With this in mind, I feel the name is fine as it is.

Reply:
OK!

Reply:
Naming things is definitely the hardest part :)